content/docs/user-guide/dvc-files.md

```diff
@@ -251,3 +266,59 @@ directory tracked by DVC. Specifically: `md5`, `etag`, or `checksum` (same as in
 
 Full <abbr>parameters</abbr> (key and value) are listed separately under
 `params`, grouped by parameters file.
+
+### Variables in `dvc.yaml`
+
+To avoid repetition, `dvc.yaml` supports variables with the following syntax.
```

Comment:
I'm also thinking we will need to split dvc.yaml from .dvc files in the guide (as a 3rd level in the main nav) so these H3s become H2s and are shown in the right-hand nav... Do you agree @shcheklein ? And if so should I do so in this same PR? Thanks

Reply:
p.s. my plan is to also add a "Stage groups" section (for loops) soon, so the doc will get pretty long... Another good reason to split from .dvc files.

p.p.s we'll just need to repeat the Outputs and Dependencies sections in both pages (.dvc file outs/deps are a little different from those in dvc.yaml anyway so that would be more correct even if there's overlap).

Reply:
Extracted to https://github.com/iterative/dvc.org/issues/2059#issuecomment-751423317